networking/networking.cpp

```diff
@@ -1281,8 +1281,8 @@ void nw_WorkReliable(uint8_t *data, int len, network_address *naddr) {
 
   reliable_socket *rsocket = NULL;
   // Check to see if we need to send a packet out.
-  if ((reliable_sockets[serverconn].status == RNF_LIMBO) &&
-      ((serverconn != -1) && (timer_GetTime() - last_sent_iamhere) > NETRETRYTIME)) {
+  if ((serverconn != -1) && (reliable_sockets[serverconn].status == RNF_LIMBO) &&
```

Comment:
could even be `serverconn >= 0` to be even safer here, but it doesn't really matter